cpplint.py

```diff
@@ -518,7 +520,27 @@ def u(x):
     xrange = range
     itervalues = dict.values
     iteritems = dict.items
+=======
+Py3k = (sys.version_info[0] == 3)
+"""A boolean to check if we are running Python3000"""
+
+try:
+    xrange(0,1)
+except NameError:
+    xrange = range
+try:
+    unicode
+except NameError:
+    basestring = unicode = str
+try:
+    long
+except NameError:
+    long = int
+>>>>>>> Added support for Python3000
+
```

Comment:
it seems your solution for xrange and unicode works better with pyflakes and py3k, so I'll copy that bit over.